Resources/Textures/_NF/Objects/Tools/maintenance-jack.rsi/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Based on maintenance-jack.dmi from NebulaSS13, modified by Whatstone",
```

Comment:
Need to reference the commit where I took it, just marking it as 413dc17c9757bf33984534a4d9b3e3db538a6285.